.ci/after_success.sh

```diff
@@ -7,13 +7,3 @@ if [[ $MATRIX_DOCKER ]]; then
 else
   coverage xml
 fi
-
-if [[ $TRAVIS ]]; then
-    codecov --flags TravisCI
-fi
```

Comment:
My idea from #5100 was `codecov --flags GitHubActions`. Why remove it instead?

Reply:
We already have this in `test.yml`, setting `GHA_X` flags via `-F`:


```yml
    - name: After success
      run: |
        .ci/after_success.sh

    - name: Upload coverage
      run: bash <(curl -s https://codecov.io/bash) -F ${{ matrix.codecov-flag }}
      env:
        CODECOV_NAME: ${{ matrix.os }} Python ${{ matrix.python-version }}
```

You can see them on `master` here:

https://codecov.io/gh/python-pillow/Pillow/commit/5c600adf4191e864568c1988cca4b045c7d05435/build

https://codecov.io/gh/python-pillow/Pillow/src/5c600adf4191e864568c1988cca4b045c7d05435/src/PIL/Image.py

Reply:
Thanks for the explanation